### ISSUE TITLE
Accessibilty issues on the Job Board page

### ISSUE THREAD

**Issue by nickytonline** [drewclem/protege#147]
## Describe the bug

There are currently accessibility violations on the job board page.

## To Reproduce

1. Install the [WAVE](https://wave.webaim.org/extension/) browser extension for.
2. Navigate to https://protege.dev/job-board or localhost:3000/job-board if your local development server is running.
2. Click on the WAVE icon in the list of extension of your browser.

![image](https://user-images.githubusercontent.com/833231/91649887-88166200-ea46-11ea-9b24-6bf3f540e8e6.png)

3. THe WAVE extension will open a side bar on the left of your browser window. Click on the Details tab to see a detailed list of accessibility violations.

![image](https://user-images.githubusercontent.com/833231/91649901-b005c580-ea46-11ea-8586-488bebd9b78d.png)

## Expected behavior

There should be no accessibilty violations on the page.

## Screenshots

![image](https://user-images.githubusercontent.com/833231/91649937-1e4a8800-ea47-11ea-895e-d1bd14e751ad.png)

## Desktop (please complete the following information):

N/A

## Smartphone (please complete the following information):

N/A

### Additional context

N/A
